backend/src/routes/criteria.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from 'express';
+import Criteria from 'models/criteria';
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  Criteria.find({}, (err, product) => {
+    if (err) res.send('Error getting all Criteria');
```

Comment:
I noticed all of the errors do not send a status. I don't think the default will be an error code. It seems like there are a few ways to do it: https://expressjs.com/en/guide/error-handling.html
1. Using `res.send`, passing in the error status: `res.status(err.statusCode).send('msg')`
2. Passing the error to the default error handler which will send a response: `next(err)`

Reply:
I set all unhandled ones to 500 for now.  I'm not sure 1 is what we want because `err` in this case is a mongo error which don't match up to HTTP status codes.